veps/sig-network/downwardapi-mac-address.md

```diff
@@ -0,0 +1,206 @@
+# VEP #127: Pass MAC address to network binding plugin sidecar
```

Comment:
This should be an extension to the original network binding pluginenhancement. Please mention this and reference to the old enhancement please.